src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+#: src/screens/Onboarding/state.ts:81
+msgid "Animals"
+msgstr "Dieren"
+
+#: src/view/com/util/post-embeds/GifEmbed.tsx:149
+msgid "Animated GIF"
+msgstr "Geanimeerde GIF"
+
+#: src/lib/moderation/useReportOptions.ts:33
+msgid "Anti-Social Behavior"
+msgstr "Asociaal gedrag"
+
+#: src/view/screens/Search/Search.tsx:347
+#: src/view/screens/Search/Search.tsx:348
+msgid "Any language"
+msgstr "Elke taal"
+
+#: src/view/com/composer/threadgate/ThreadgateBtn.tsx:49
+msgid "Anybody can interact"
+msgstr "Iedereen kan reageren"
+
+#: src/view/screens/LanguageSettings.tsx:94
+msgid "App Language"
+msgstr "App-taal"
+
+#: src/view/com/modals/AddAppPasswords.tsx:138
+msgid "App Password names can only contain letters, numbers, spaces, dashes, and underscores."
+msgstr "App-wachtwoordnamen mogen alleen letters, cijfers, spaties, streepjes en onderstrepingstekens bevatten."
+
+#: src/view/com/modals/AddAppPasswords.tsx:103
+msgid "App Password names must be at least 4 characters long."
+msgstr "App-wachtwoordnamen moeten minimaal 4 tekens lang zijn."
+
+#: src/Navigation.tsx:285
+#: src/view/screens/AppPasswords.tsx:197
+#: src/view/screens/Settings/index.tsx:673
+msgid "App Passwords"
+msgstr "App-wachtwoorden"
+
+#: src/view/screens/AppPasswords.tsx:232
+msgid "App password deleted"
+msgstr "App-wachtwoord verwijderd"
+
+#: src/view/screens/Settings/index.tsx:664
+msgid "App password settings"
+msgstr "App-wachtwoordinstellingen"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:148
+#: src/components/moderation/LabelsOnMeDialog.tsx:151
+msgid "Appeal"
+msgstr "Beroep"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:243
+msgid "Appeal \"{0}\" label"
+msgstr "Beroep tegen \"{0}\"-label"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:233
+#: src/screens/Messages/components/ChatDisabled.tsx:91
+msgid "Appeal submitted"
+msgstr "Beroep ingediend"
+
+#: src/screens/Messages/components/ChatDisabled.tsx:51
+#: src/screens/Messages/components/ChatDisabled.tsx:53
+#: src/screens/Messages/components/ChatDisabled.tsx:99
+#: src/screens/Messages/components/ChatDisabled.tsx:101
+msgid "Appeal this decision"
+msgstr "Beroep aantekenen tegen deze beslissing"
+
+#: src/screens/Settings/AppearanceSettings.tsx:89
+#: src/view/screens/Settings/index.tsx:485
+msgid "Appearance"
+msgstr "Weergave"
+
+#: src/Navigation.tsx:325
+msgid "Appearance Settings"
+msgstr "Weergave-instellingen"
+
+#: src/view/screens/Settings/index.tsx:476
+msgid "Appearance settings"
+msgstr "Weergave-instellingen"
+
+#: src/screens/Feeds/NoSavedFeedsOfAnyType.tsx:47
+#: src/screens/Home/NoFeedsPinned.tsx:93
+msgid "Apply default recommended feeds"
+msgstr "Standaard aanbevolen feeds toepassen"
+
+#: src/components/dialogs/MutedWords.tsx:433
+msgid "Are you sure?"
+msgstr "Weet u het zeker?"
+
+#: src/view/com/composer/Composer.tsx:532
+msgid "Are you sure you'd like to discard this draft?"
+msgstr "Weet u zeker dat u dit concept wilt laten vervallen?"
```

Comment:
Discard is intetionally translated by "vervallen" instead of "verwijderen" of "weggooien" because it in my opinion more closely matches the semantics of a draft being discarded.

Reply:
In case there are other use cases such as for users migrating from other platforms or standards please let me know.

Reply:
If I find consensus among other platforms, I'll create an issue or open a PR :)